app/src/main/java/ru/flicksbox/user/data/UserRepository.kt

```diff
@@ -0,0 +1,21 @@
+package ru.flicksbox.user.data
+
+import kotlinx.coroutines.flow.*
+import ru.flicksbox.data.Data
+import ru.flicksbox.user.domain.UserEntity
+import ru.flicksbox.user.domain.UserRepository
+import ru.flicksbox.network.BaseDTO
+
+class UserRepositoryImpl(
+    private val userService: UserService,
+) : UserRepository {
+    override fun getUser(cookie: String): Flow<Data<UserEntity>> =
+        flow { emit(userService.getUser(cookie)) }
+            .map { user -> user.body!!.toDomain() }
```

Comment:
it's better to avoid force unwrap, in this case I suggest to write sth like this, and throw error by yourself
```kotlin
val body = user.body ?: throw <some error>
body.toDomain()
```